test/commands/environment/tail-log.test.js

```diff
@@ -47,3 +47,14 @@ test('tail-log - config', async () => {
   await expect(mockSdk.tailLog.mock.calls.length).toEqual(1)
   await expect(mockSdk.tailLog).toHaveBeenCalledWith('5', '17', 'author', 'aemerror', process.stdout)
 })
+
+test('tail-log - should retry 5 times and throw error', async () => {
+  setCurrentOrgId('good')
```

Comment:
Can we mock something other than 'good' ?

Reply:
I see we use this in other tests